.gitignore

```diff
@@ -97,6 +97,7 @@ _build
 /lex/parser.output
 
 /manual/src/cmds/warnings-help.etex
+/manual/src/html_processing/src/common.ml
```

Comment:
This needs to be removed by the `distclean` target in `manual/src/html_processing/Makefile`